Examples/SubscriberExample/Resources/Info.plist

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>ABLY_API_KEY</key>
```

Comment:
What is this `ABLY_API_KEY`? It doesn't seem related to anything else in this PR.

Reply:
Ah, I see it's just moved.